frontend/src/pages/AdminPanel.js

```diff
@@ -0,0 +1,1253 @@
+  };
+  
+  const closeSuggestionModal = () => {
+    setSelectedSuggestion(null);
+    setIsSuggestionModalOpen(false);
+  };
+  
+  const openSuggestionDeleteConfirm = (suggestion) => {
+    setSelectedSuggestion(suggestion);
+    setIsSuggestionDeleteConfirmOpen(true);
+  };
+  
+  const closeSuggestionDeleteConfirm = () => {
+    setSelectedSuggestion(null);
+    setIsSuggestionDeleteConfirmOpen(false);
+  };
+  
+  const deleteSuggestion = () => {
+    axiosClient
+      .delete(`/suggestions/${selectedSuggestion.id}`)
+      .then(() => {
+        setSuggestions((prevSuggestions) =>
+          prevSuggestions.filter((s) => s.id !== selectedSuggestion.id)
+        );
+        closeSuggestionDeleteConfirm();
+      })
+      .catch((error) => {
+        console.error('Error deleting suggestion:', error);
+      });
+  };
+
+  return (
+    <div className="min-h-screen bg-lightGrayish relative">
+      <Header />
+      <div className="flex">
+        {/* Panel boczny */}
+        <aside className="w-1/5 min-h-[calc(100vh-6rem)] bg-darkGreen text-white p-4 flex flex-col items-center">
+          <h2 className="text-lg font-bold mb-4 text-center">Panel Administratora</h2>
+          <ul className="space-y-4 w-full">
+            <li>
+              <button onClick={() => setActiveTab('users')} className={`w-full text-left py-2 px-4 rounded ${activeTab === 'users' ? 'bg-oliveGreen' : 'hover:bg-oliveGreen'}`}>
+                Użytkownicy
+              </button>
+            </li>
+            <li>
+              <button onClick={() => setActiveTab('kebabs')} className={`w-full text-left py-2 px-4 rounded ${activeTab === 'kebabs' ? 'bg-oliveGreen' : 'hover:bg-oliveGreen'}`}>
+                Kebaby
+              </button>
+            </li>
+            <li>
+              <button onClick={() => setActiveTab('suggestions')} className={`w-full text-left py-2 px-4 rounded ${activeTab === 'suggestions' ? 'bg-oliveGreen' : 'hover:bg-oliveGreen'}`}>
+                Sugestie
+              </button>
+            </li>
+          </ul>
+        </aside>
+
+        {/* Main */}
+        <main className="flex-1 p-6">
+        {!activeTab && (
+            <div className="flex items-center justify-center h-[calc(100vh-6rem)]">
+              <h1 className="text-2xl font-bold text-darkGreen text-center">
+                Witaj w Panelu Administratora
+              </h1>
+            </div>
+        )}
+
+        {/* Tabela users */}
+        {activeTab === 'users' && (
+        <div className="w-full">
+          <h1 className="text-2xl font-bold text-darkGreen mb-6">Użytkownicy</h1>
+          <table className="w-full table-auto bg-white rounded-lg shadow-md">
+            <thead className="bg-darkGreen text-white">
+              <tr>
+                <th className="px-4 py-2 text-center">Użytkownik</th>
+                <th className="px-4 py-2 text-center">Email</th>
+                <th className="px-4 py-2 text-center">Hasło</th>
```

Comment:
I think there's no need to keep password in users table since it's just "*****" in every case, editing it is a good idea tho